User-Help/DeletedPostHelp

```diff
@@ -0,0 +1,15 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| articles | Site Information |
```

Comment:
URL slug probably isn't articles?

Reply:
I copied the formatting, meant to update it, and totally forgot. Doh!